src/posix/platform/settings.cpp

```diff
@@ -54,9 +54,12 @@
 #include "common/code_utils.hpp"
 #include "common/encoding.hpp"
 
+#include "settings.hpp"
```

Comment:
`settings.hpp` should be at the top according to your code style?

Reply:
Indeed. Though this case a little special: `setting.cpp` doesn't implement`setting.hpp`, it only implement one method. I'll update it.